configs/openmrs_config/globalproperties/i18n.xml

```diff
@@ -4,5 +4,10 @@
             <property>locale.allowed.list</property>
             <value>en, km</value>
         </globalProperty>
+        <globalProperty>
+        <property>default_locale</property>
```

Comment:
Some slight formatting needed here @kazlaw.

Reply:
Noted let me work on it.

Reply:
@Ruhanga , I have made adjustments. Is the latest commit good enough?